### PR TITLE
Improve logs on KVMHAVMActivityChecker

### PR DESCRIPTION
**Pull request by GutoVeronezi**
### Description
This PR intends to improve logging in the class `KVMHAVMActivityChecker` to facilitate troubleshooting.

### Types of changes
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functioanlity)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale
- [ ] Major
- [x] Minor

### How Has This Been Tested?
It has been tested locally on a test lab.

### CODE REVIEW

**Review by DaanHoogland — APPROVED** [review]

File: plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/KVMHAVMActivityChecker.java [hunk 1]
```diff
@@ -52,11 +52,14 @@ public Boolean checkingHB() {
...
         OutputInterpreter.OneLineParser parser = new OutputInterpreter.OneLineParser();
+
         String result = cmd.execute(parser);
-        LOG.debug("KVMHAVMActivityChecker pool: " + nfsStoragePool._poolIp);
-        LOG.debug("KVMHAVMActivityChecker result: " + result);
-        LOG.debug("KVMHAVMActivityChecker parser: " + parser.getLine());
-        if (result == null && parser.getLine().contains("DEAD")) {
+        String parsedLine = parser.getLine();
+
+        LOG.debug(String.format("Checking HB with KVMHAVMActivityChecker [{command=\"%s\", result: \"%s\", log: \"%s\", pool: \"%s\"}].", cmd.toString(), result, parsedLine, nfsStoragePool._poolIp));
```
Comment (GabrielBrascher) [inline plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/KVMHAVMActivityChecker.java:59]: I assuming that HB should be HA:
1. Checking **HB** with KVMHAVMActivityChecker ...
2. Checking **HA** with ...

**Review by GabrielBrascher — CHANGES_REQUESTED** [review]
@GutoVeronezi thanks for the PR; can you please check the raised point?

File: plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/KVMHAVMActivityChecker.java (hunk 1 above)
Comment (GabrielBrascher) [reply plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/KVMHAVMActivityChecker.java:59]: Never mind, it is the Heart Beat, right?

I would suggest a different name then as HB might not mean much.

Comment (GutoVeronezi) [reply plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/KVMHAVMActivityChecker.java]: @GabrielBrascher done.

**Review by GabrielBrascher — APPROVED** [review]
LGTM

File: plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/KVMHAVMActivityChecker.java [hunk 2]
```diff
@@ -52,11 +52,14 @@ public Boolean checkingHB() {
         cmd.add("-t", String.valueOf(String.valueOf(System.currentTimeMillis() / 1000)));
         cmd.add("-d", String.valueOf(suspectTimeInSeconds));
         OutputInterpreter.OneLineParser parser = new OutputInterpreter.OneLineParser();
+
```
Comment (sureshanaparti) [inline plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/KVMHAVMActivityChecker.java:55]: @GutoVeronezi may be you can also improve this method name,  `checkingHB()` => _checkingHeartBeat()_

Comment (GabrielBrascher) [reply plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/KVMHAVMActivityChecker.java:55]: **+1** HB is not really telling much

**Review by sureshanaparti — APPROVED** [review]

**Comment by GutoVeronezi** [comment]
@DaanHoogland @GabrielBrascher @sureshanaparti is there anything else to do?

**Comment by DaanHoogland** [comment]
@GutoVeronezi on first sight the error in the smoke test run seems related to the changes, though the stack trace in the result file for that test does not. I'm re-running, if it fails again please have a look.

**Pull request merged**